source/administration/upgrade.rst

```diff
@@ -99,6 +99,7 @@ Location of your local storage directory
    .. code-block:: sh
 
      sudo mv mattermost/plugins/ mattermost/plugins~
+	 sudo mv mattermost/client/plugins/ mattermost/client/plugins~

```

Comment:
Please left align the commands.
![screen shot 2019-02-18 at 5 01 06 pm](https://user-images.githubusercontent.com/5334504/52939331-dd2bdf00-339e-11e9-9c5a-2660a47b49a0.png)